netket/operator/der_local_values_jax.py

```diff
@@ -75,7 +74,7 @@ def _local_values_and_grads_notcentered_kernel(pars, vp, mel, v, logpsi):
 
 def _der_local_values_notcentered_impl(op, machine, v, log_vals):
     sections = _np.empty(v.shape[0], dtype=_np.int32)
-    v_primes, mels = op.get_conn_flattened(v, sections, pad=True)
+    v_primes, mels = op.get_conn_flattened(v._value, sections, pad=True)
```

Comment:
is using _value safe though? This looks like an internal attribute that might not be always consistent?

Reply:
`._value` is not stable API according to Jax, but it's used internally for the exact same purpose in the Jax-supported wrappers of DLPack. I doubt there is another way to do this without copying.

Reply:
I'd keep this like that for the time being, and if Jax stabilises the api we can upgrade too.